axiom/tokens_test.go

```diff
@@ -0,0 +1,256 @@
+package axiom
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var tokenTime = time.Now()
```

Comment:
Please don't use `time.Now()`. The implications this introduces are visible throughout the rest of the file.

You have to `sprintf` your responses and deal with truncation and timezones. Just use `testhelper.MustTimeParse` are use static strings, e.g. https://github.com/axiomhq/axiom-go/blob/main/axiom/datasets_test.go#L250.

Reply:
Thanks! Made that change